crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -290,6 +298,36 @@ def ha_config_check
             end
           end
         end
+
+        deployment_supported =
+          case clusters_roles.length
+          when 0
+            # no clusters, no point complaining as this will be detected by other prechecks
+            true
+
+          when 1
+            # everything on one cluster = no problem
+            true
+
+          when 2
+            # neutron-network in separate cluster
```

Comment:
Could you please write some example values of `clusters_roles` and `roles_clusters` here?